src/components/timers/Countdown.tsx

```diff
@@ -1,3 +1,85 @@
-const Countdown = () => null;
+import { useState } from 'react';
+import { PlayPauseButton, FastForwardButton, ResetButton } from '../generic/Button';
+import { Input } from '../generic/Input';
+import { DisplayTime } from '../generic/DisplayTime';
+import { convertToMs } from '../../utils/helpers';
+
+const Countdown = () => {
+    const [countdownMinValue, setCountdownMinValue] = useState(0);
+    const [countdownSecValue, setCountdownSecValue] = useState(0);
+    const [countdownTime, setCountdownTime] = useState(0);
+    const [isCountdownRunning, setIsCountdownRunning] = useState(false);
+    const [intervalId, setIntervalId] = useState(0);
```

Comment:
you want to use a useRef for the intervalId instead of useState to avoid race conditions. If someone triggers the setter a bunch of times rapidly, it's not clear that the clearInterval call will get the correct intervalId. You can also avoid unnecessary renders by using useRef.